### PR TITLE
End leftover YAML files (SGE only for now)

### PR DESCRIPTION
**Pull request by sfranky**

### CODE REVIEW

File: qtop.py [hunk 1]
```diff
@@ -1225,6 +1226,8 @@ def get_yaml_files(scheduler, filenames):
 
 def get_filenames_commands():
     d = dict()
+    # date = time.strftime("%Y%m%d")
+    # fn_append = "_" + str(date) if not options.SOURCEDIR else ""
```
Comment (fgeorgatos) [inline qtop.py]: ?

File: qtop.py [hunk 2]
```diff
@@ -1357,6 +1368,26 @@ def check_python_version():
         sys.exit(1)
 
 
+def take_care_of_old_yaml_files(filepath):
+    """
+    deletes older yaml files in savepath directory. Experimental and untested!!
+    USE AT OWN RISK
+    """
+    """
```
Comment (fgeorgatos) [inline qtop.py]: ?

File: qtopconf.yaml [hunk 3]
```diff
@@ -235,3 +236,5 @@ occupancy_column_order:
   - Node state
   - queue name
 horizontal_separator: '|'
+
+auto_delete_old_yaml_files_after: 1 # (hours)
```
Comment (fgeorgatos) [inline qtopconf.yaml]: after_few_hours?

File: qtop.py (hunk 2 above)
Comment (sfranky) [inline qtop.py]: this has been fixed in next PR

**Comment by fgeorgatos** [comment]
LGTM!

**Pull request merged**